vuepress/docs/next/docs/getting-started/concepts-overview.md

```diff
@@ -1,164 +1,102 @@
 
-The following is an overview of the members of an Entando cluster and their roles.
+## Entando Cluster Elements
 
-### Architecture -Members of the Cluster
+The following is an overview of the elements comprising an Entando Cluster.
 
-![Entando Cluster Architecture Diagram](./img/entando-architecture-v1.5.svg)
+### Architecture - Members of the Cluster
 
+Below is a diagram of an Entando Cluster and depicts how the various elements interact with one another. Entando deploys this infrastructure on Kubernetes using the Entando Operator and controllers. Each element is associated with a custom resource definition (CRD) file, if applicable.
 
-This picture of an Entando cluster shows how the various citizens interact with each other. Most of the citizens have a Custom Resource Definition
-file associated with them. They are deployed on Kubernetes, through the
-Entando operator and controllers.
+![Entando Cluster Architecture Diagram](./img/entando-architecture-v1.5.svg)
 
 #### EntandoApp
-
-The EntandoApp is composed of three parts:
-
-1.  **Entando App Builder**: the user interface to build and customize an application.
-
-2.  **Entando App Engine**: the backend APIs providing access to 
-    services.
-
-3.  **Entando Component Manager (ECM)**: the service integrating the Local Hub into the App Builder providing bundle management for deploying and versioning components.
-
-The interaction between these three components, and the rest of the cluster, use the authorization/authentication features provided by Keycloak.
-
+The EntandoApp is comprised of the Entando App Builder, the Entando App Engine and the Entando Component Manager. Keycloak-based authorization and authentication enable these to interact with each other and with other cluster elements.
 #### Entando Component Manager
+The purpose of the [Entando Component Manager](../compose/ecm-overview.md) is to:
+- Provide the functionality to deploy and install micro frontends and widgets
+- Manage the connections between an application and the installed microservices
 
-The [Entando Component Manager](../compose/ecm-overview.md) lists the EntandoDeBundles accessible from the EntandoApp and provides the install/uninstall/upgrade/downgrade services to manage a bundle. The K8s service communicates with the cluster to provide these services. It is the only service, other than the operator, that can interact with the cluster and the required custom resources. 
+It integrates the Local Hub into the App Builder to list the Entando Bundles accessible from the EntandoApp, then provisions bundle management services such as install, uninstall, upgrade, downgrade and versioning. The Entando Kubernetes integration service communicates with the cluster to supply these bundle services and is the only service, other than the Entando Operator, that can interact with the cluster and custom resources. 
```

Comment:
no. to provision something is to say it's made available beforehand. these capabilities are built in to the functionality and activated, not newly available. should be "provisioning" instead of "then provisions"

Reply:
provides means to supply or make available, to make something available to, and to have as a condition. Provisioning is the act of providing but agree to disagree.

Reply:
actually, in this context, provisioning is correct, and providing is not.
https://www.redhat.com/en/topics/automation/what-is-provisioning
<img width="819" alt="Screen Shot 2022-11-29 at 8 15 25 AM" src="https://user-images.githubusercontent.com/93157274/204583455-a2891348-9144-4d51-9477-4e706a748550.png">